drivers/avr/serial.c

```diff
@@ -20,50 +20,83 @@
 
 #ifdef SOFT_SERIAL_PIN
 
-#    if defined(__AVR_ATmega16U2__) || defined(__AVR_ATmega32U2__) || defined(__AVR_ATmega16U4__) || defined(__AVR_ATmega32U4__)
-// if using ATmegaxxU4 I2C, can not use PD0 and PD1 in soft serial.
+#    if (defined(__AVR_AT90USB646__) || defined(__AVR_AT90USB647__) || defined(__AVR_AT90USB1286__) || defined(__AVR_AT90USB1287__) || defined(__AVR_ATmega16U2__) || defined(__AVR_ATmega32U2__) || defined(__AVR_ATmega16U4__) || defined(__AVR_ATmega32U4__))
+// if using ATmega32U4 I2C, can not use PD0 and PD1 in soft serial.
```

Comment:
```suggestion
// if using ATmega16/32U4 I2C, can not use PD0 and PD1 in soft serial.
```
Should be same for 16U4

Reply:
it's also the same for at90usb.  

It's not true for the atmega32a, but that doesn't support those pins in general, but does support UART on those pins. 

Would it be best to just remove reference to the controller, in general?

Reply:
Ah yeah. Updated.